extensions/wikia/TemplateClassification/scripts/TemplateClassificationInView.js

```diff
@@ -14,6 +14,15 @@ define('TemplateClassificationInView', ['jquery', 'mw', 'wikia.nirvana', 'Templa
 		function init() {
 			$typeLabel = $('.template-classification-type-text');
 			templateClassificationModal.init(getType, sendClassifyTemplateRequest);
+
+			$('.template-classification-edit').on('click', function() {
+				tracker.track({
+					trackingMethod: 'both',
```

Comment:
'both' is removed from our code. Use 'analytics'.
per email from @harnash "- analytics - send data to Google Analytics and internal warehouse (old ‘both’ method and effectively also ‘ga’)"